app/src/store/views/appView.ts

```diff
@@ -36,6 +36,9 @@ export default class AppView {
   goTo(route: string) {
     if (this._store.router.location.pathname !== route) {
       this._store.router.push(route);
+
+      // Check for updates to subservers when navigating between pages
+      this._store.subServerStore.fetchStatus();
```

Comment:
I included this in case the node restarts while the UI is still open.

This often happens for me during development. For example, while testing the subserver changes I often leave my lightning-terminal browser window open while I make changes to my `regtest.yml` file and restart the containers. Without this change my subserver status won't update unless I manually refresh the page. Maybe this isn't likely to happen in a real-world scenario :thinking:

Reply:
Yea, I don't think it's a good trade-off to have all users fetch so aggressively just to streamline this rare case for average users. The streaming RPCs also disconnect when the node is restarted, so users need to reload the page to regain full functionality of the frontend app.